File: LV0/다음에 올 숫자/solution.py
def solution(common):
    answer = 0
    
    if common[1]-common[0] == common[2]-common[1]:
        answer = common[-1] + (common[1]-common[0])
    else :
        answer = common[-1] * (common[1]/common[0])
    return answer

#https://school.programmers.co.kr/learn/courses/30/lessons/120924?language=python3 

File: LV0/다음에 올 숫자/solution.js
function solution(common) {
    var answer = 0;
    if((common[1]-common[0]) ==(common[2]-common[1])){
        console.log('?')
        var dungcha = common[1]-common[0]
        answer = common.at(-1)+dungcha
    }else{
        var dungbi = common[1]/common[0]
        answer = common.at(-1)*dungbi
    }
    return answer;
}

//https://school.programmers.co.kr/learn/courses/30/lessons/120924